source/ui/Image.js

```diff
@@ -199,10 +231,10 @@
 			this.tag = this.sizing ? 'div' : 'img';
 			this.addRemoveClass('sized', !!this.sizing);
 			if (was) {
-				this.removeClass(was);
+				this.removeClass('enyo-'+was);
 			}
 			if (this.sizing) {
-				this.addClass(this.sizing);
+				this.addClass('enyo-'+this.sizing);
```

Comment:
My mistake, and apologies. These should not have the "enyo-" prefix. I was remembering incorrectly.